### ISSUE TITLE
"Connects" to a 2016 TV w/ Tizen... but doesn't work. Help?

### ISSUE THREAD

**Issue by inzlabs** [kyleaa/homebridge-samsungtv2016#10]
I will jump into this, sadly with a problem but no much to add on how to find a solution. I have a: 

```
  "name": "[TV] Samsung 6 Series (50)",
  "version": "2.1.0",
  "device": {
    "type": "Samsung SmartTV",
    "modelName": "UN50KU6300",
    "ip": "192.168.1.13",
    "firmwareVersion": "Unknown",
    "name": "[TV] Samsung 6 Series (50)",
    "resolution": "3840x2160",
    "countryCode": "US",
    "msfVersion": "2.1.0",
    "smartHubAgreement": "true",
    "OS": "Tizen"
```

Once this plugin installed, I can see it in the Home app as an accessory, and also on the Homebridge. It stays the TV is on. When I try to turn it off, instantaneously the TV asks me for an allow permission. I allow. But the switch off, doesn't work. 

At the same time, I can't change the location of the TV (to living room), or the name of the TV on the Home app. 

Also using another app: Koogeek Home brings this error: https://imgur.com/gallery/v3xBg

If anyone has an idea on how to fix this, I am all ears and eyes.

ps. I researched all other Samsung TV plugins, this work better with 2016 tvs. Thank you for your work.

ps 2. This is the log from booting up Homebridge, no errors so far:
```
[12/30/2016, 6:18:58 AM] Loaded plugin: homebridge-nest
[12/30/2016, 6:18:58 AM] Registering platform 'homebridge-nest.Nest'
[12/30/2016, 6:18:58 AM] ---
[12/30/2016, 6:18:59 AM] Loaded plugin: homebridge-samsungtv2016
[12/30/2016, 6:18:59 AM] Registering accessory 'homebridge-samsungtv.SamsungTV2016'
[12/30/2016, 6:18:59 AM] ---
[12/30/2016, 6:18:59 AM] Loaded config.json with 1 accessories and 1 platforms.
[12/30/2016, 6:18:59 AM] ---
[12/30/2016, 6:18:59 AM] Loading 1 platforms...
[12/30/2016, 6:18:59 AM] Initializing Nest platform...
[12/30/2016, 6:18:59 AM] Fetching Nest devices.
[12/30/2016, 6:18:59 AM] Loading 1 accessories...
[12/30/2016, 6:18:59 AM] [TV Living room] Initializing SamsungTV2016 accessory...

```

**Comment by inzlabs** [comment]
Adding a little more of light to this, I found out that when the TV is off, without Ethernet. Via the Home app, it can be turned On, but then this triggers the Homebridge (on a raspberry pi) to kill itself. In any scenario I could turn the TV off.

This is part of the log after turning the TV on:
```
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: connect ETIMEDOUT 192.168.1.13:8001
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1087:14)
```

**Comment by ToddGreenfield** [comment]
I have the exact same TV. I installed the plugin and am able to turn the TV on (if its off).
I am unable to turn the TV off though. The syslog shows what appears to be things working without error, but it doesnt turn off-

Dec 31 10:57:34 homepi homebridge[21127]: [12/31/2016, 10:57:34 AM] [Bedroom TV] TV API is active
Dec 31 10:57:38 homepi homebridge[21127]: [12/31/2016, 10:57:38 AM] [Bedroom TV] received on command: 0
Dec 31 10:57:38 homepi homebridge[21127]: [12/31/2016, 10:57:38 AM] [Bedroom TV] sending power key
Dec 31 10:57:38 homepi homebridge[21127]: [12/31/2016, 10:57:38 AM] [Bedroom TV] successfully powered off tv
Dec 31 10:57:42 homepi homebridge[21127]: [12/31/2016, 10:57:42 AM] [Bedroom TV] received on command: 1
Dec 31 10:57:42 homepi homebridge[21127]: [12/31/2016, 10:57:42 AM] [Bedroom TV] TV API is active
Dec 31 10:57:42 homepi homebridge[21127]: [12/31/2016, 10:57:42 AM] [Bedroom TV] sending power key
Dec 31 10:57:42 homepi homebridge[21127]: [12/31/2016, 10:57:42 AM] [Bedroom TV] successfully powered on tv

I am also able to move the location from one room to another without issue.
I did notice in my router when verifying the IP of the TV that it had multiple listings with different IPs (all with the same MAC) so I gave it a perm reservation.

I am also running wireless-

**Comment by kyleaa** [comment]
The three comments on this PR look like duplicates of #6, #7, and #11, so I'm closing this ticket.

I believe the issue with power off may be resolved, but thats tracking in #11.
